src/Service.Host/client/src/components/parts/PartTemplate.js

```diff
@@ -0,0 +1,534 @@
+                                    { id: 'A', displayText: 'Assembly' },
+                                    { id: 'P', displayText: 'Phantom' }
+                                ]}
+                                fullWidth
+                                value={partTemplate.bomType}
+                                onChange={handleFieldChange}
+                            />
+                        </Grid>
+
+                        <Grid item xs={3}>
+                            <Dropdown
+                                label="Rm fg"
+                                propertyName="rmfgCode"
+                                items={[
+                                    { id: 'R', displayText: 'Raw Material' },
+                                    { id: 'F', displayText: 'Finished Good' }
+                                ]}
+                                fullWidth
+                                value={partTemplate.rmfgCode}
+                                onChange={handleFieldChange}
+                            />
+                        </Grid>
+
+                        <Grid item xs={6} />
+                        <Grid item xs={6}>
+                            <Dropdown
+                                label="Assembly Technology"
+                                propertyName="assemblyTechnology"
+                                items={assemblyTechnology.map(c => ({
+                                    id: c.name,
+                                    displayText: c.description
+                                }))}
+                                fullWidth
+                                allowNoValue
+                                value={partTemplate.assemblyTechnology}
+                                onChange={handleFieldChange}
+                            />
+                        </Grid>
+                        <Grid item xs={3}>
+                            <Dropdown
+                                fullWidth
+                                label="Pareto Code"
+                                propertyName="paretoCode"
+                                items={[
+                                    { id: 'A', displayText: 'A - BOUGHT IN, TOP 80% OF SPEND' },
+                                    { id: 'B', displayText: 'B - BOUGHT IN, NEXT 15% OF SPEND' },
+                                    { id: 'C', displayText: 'C - BOUGHT IN, LEAST 5% OF SPEND' },
+                                    { id: 'D', displayText: 'D - BOUGHT IN. NO USAGE' },
+                                    { id: 'E', displayText: 'E - LINN PRODUCED PARTS' },
+                                    { id: 'L', displayText: 'L - LOEWE TVS AND ACCESSORIES' },
+                                    { id: 'R', displayText: 'R - ALL RECORDS, TAPES AND CDS' },
+                                    { id: 'U', displayText: 'U - UNSOURCED COMPONENTS' },
+                                    { id: 'X', displayText: 'X - PARTS OUTWITH PARETO CODES' }
```

Comment:
This one probably could do with a real database lookup for the results. We spoke about doing that in your next PR already though so feel free to merge as is